jetty-servlets/src/main/java/org/eclipse/jetty/servlets/DoSFilter.java

```diff
@@ -1503,13 +1368,13 @@ public Action onRequestOverLimit(HttpServletRequest request, OverLimit overlimit
             switch (action)
             {
                 case REJECT:
-                    LOG.warn("DOS ALERT: Request rejected ip={}, overlimit={}, session={}, user={}", request.getRemoteAddr(), overlimit, request.getRequestedSessionId(), request.getUserPrincipal());
+                    LOG.warn("DOS ALERT: Request rejected ip={}, overlimit={}, user={}", request.getRemoteAddr(), overlimit, request.getUserPrincipal());
```

Comment:
Nit: denial of service is usually stylized as `DoS`, while `DOS` refers to everyone's favorite operating system of the 80's.